CHANGELOG.md

```diff
@@ -83,6 +83,7 @@ Querying with using `{__mimir_storage__="ephemeral"}` selector no longer works.
 ### Documentation
 
 * [ENHANCEMENT] Document migration from microservices to read-write deployment mode. #3951
+* [BUGFIX] Added indentation to azure an swift backend definition.
```

Comment:
@bartoszcisek Do you mind to add this PR number (`#4263`) at the end of the CHANGELOG entry, please? Thanks!